### DISCUSSION TITLE
Auto-Swiper

### DISCUSSION THREAD

**Discussion by surtic86** [Glench/ExtPay#56] [Show and tell]
So i just saw that there was the Category to show off and tell about what i have don. So here i am and hope to get Feedback from you what do you think about it. Maybe we can help us each out with tips what we can do better.

Since 1Q of 2020 i have created an Auto Swiper for Tinder and expended it over the Years with new Features and other Dating Sites. And now since  the end of Mai i have implemented Extensionpay.

Website: https://www.auto-swiper.ch/
Indiehackers: https://www.indiehackers.com/product/auto-swiper

Chrome Webstore: https://chrome.google.com/webstore/detail/auto-swiper/knhjggkkeoamjgogdmmefekofllomhea?hl=de&authuser=0
Microsoft Store: https://microsoftedge.microsoft.com/addons/detail/auto-swiper/lnenhbhcpcjfnfijcbnaocgmgcjimdgi

Firefox: Waiting on Manifest 3
Safari: Waiting on Manifest 3 and Extensionpay Support

**Comment by burkybang** [comment]
Thanks for sharing!

**Comment by surtic86** [comment]
@burkybang what are you working on? 

Btw. made a new Video for Presentation 

https://www.youtube.com/watch?v=uI8a027d55Y

**Reply by burkybang** [reply]
I have multiple extensions, but this is the one that I use ExtensionPay with: https://burkybang.com/SuiteAdvanced
